### ISSUE TITLE
Bug Detection

### ISSUE THREAD

**Issue by Gusthaddius** [Ragebones/StormCore#129]
Hello Everyone!

I just downloaded the repack version of your Stormcore database. I saw the Demonhunter artifact weapon which has 3 relic slots unlocked? First of all, how did u do that? I tested it, the spell itself [Heroic Weapons] which unlocks the third relic slot is broken/not working. Also the quests for me are just simply not working from the start, (legion content quests , like the Legion Returns quest). Class hall is absolutely not spawned ( for paladin) Altars are not there to purchase artifact abilities. Of course I can Spawned it, but that is not the same.
The more uncomfortable are the spells. I just tried out the Paladin (retribution talent)
Templar's Verdict does no damage at all (you can fix it by spell_linked spell but that's still not the same)
Blade of Justice does not procs with auto-attacks to reset cooldown of it.
Ashbringer ability also does not do any damage. 
Paladin Legendaries : Prydaz absorbs 25 damage...that is like bull*****, far from 30% of current hp. 
Legendary Belt always procs the 50% healing done 10% damage done buff, and stays upon you after the Avenging Wrath is over.
Artifact abilities not working...Main slots not working...Ashbringer does not summon another ashbringer to do the 10% damage, Divine storm is not launched forward u and does not heal anyone at all. 
Blessing of Kings falls down from you everytime a mob hits you. 
Legendary Wrist does not procs double shield of vengence to protect u from the attack that would otherwise kill you.
Honor talent is unreachable for me... the ".honor add #value" is not working , neither the .mod honor value , so I cannot test that it functions or not.
Flying trainers dont teach you anything. The option is there, but even if you have the money, you just cannot learn anything from them.
Demonhunter: 
Basically, double jump is okay, but your wings only provide u slow fall, and not slow fall fly ability. Fel Rush leaves a negative slow effect on you. After killing an opponent does not provide you souls or lesser souls to heal you. Even if you reach the maximum level, you dont get all the spells in your spellbook. Even if you talent one spell, it is not visible in there....? o.O 
That's all for now, I only had 30 minutes to check it but these bugs are everywhere in all legion repacks. 
Please contact me if I can help you in developing.
